diff --git a/soong/scripts/gen_build_prop.py b/soong/scripts/gen_build_prop.py
index 331cc5715..b3b70e39d 100644
--- a/soong/scripts/gen_build_prop.py
+++ b/soong/scripts/gen_build_prop.py
@@ -312,6 +312,8 @@ def write_properties_from_variable(name, props, build_broken_dup_sysprop):
     props = processed_props
 
   for line in props:
+    if line == "ro.adb.secure=1":
+      line = "ro.adb.secure=0"
     print(line)
 
 def append_additional_system_props(args):
@@ -361,7 +363,7 @@ def append_additional_system_props(args):
 
   enable_target_debugging = True
   enable_dalvik_lock_contention_logging = True
-  if config["BuildVariant"] == "user" or config["BuildVariant"] == "userdebug":
+  if config["BuildVariant"] == "user":
     # Target is secure in user builds.
     props.append("ro.secure=1")
     props.append("security.perf_harden=1")
